subsystem/intake.py

```diff
@@ -46,3 +47,19 @@ def detect_coral(self) -> bool:
         """
         self.coral_in_intake = self.sensor.getVoltage()
         return self.coral_in_intake
+    
+    def get_current(self) -> int:
+        pass
+
+    def getAppliedOutput():
+        pass
+    
+    def periodic(self) -> None:
+        if config.NT_INTAKE:
+            table = ntcore.NetworkTableInstance.getDefault().getTable('intake')
```

Comment:
Let's create an update_table method that gets called here. Just because that is how we do things...